nifi-api/src/main/java/org/apache/nifi/flow/VersionedProcessor.java

```diff
@@ -44,6 +44,11 @@
     private Set<String> autoTerminatedRelationships;
     private ScheduledState scheduledState;
 
+    private Integer retryCounts;
+    private Set<String> retriedRelationships;
```

Comment:
Similar to how we have the `autoTerminatedRelationships` property here and the `autoTerminate` property as part of the RelationshipDTO, we should add a `retry` boolean property to the RelationshipDTO.

Reply:
Yeah that's a good call @sardell. One of those fun things that's harder to catch when developing the backend without the front end :) @timeabarna can you add that in? RelationshipDTO should have a `Boolean retry` with getter/setter methods, and the DtoFactory should populate that.